File: beecrowd_urionlinejudge/uri1008Salario.cpp
#include <iostream>
#include<iomanip>
#include <stdio.h>

using namespace std;

int main() {

int NumeroFuncionario,HorasTrabalhadas;
double ValorHora,Salario;

cin >>NumeroFuncionario >>HorasTrabalhadas >>ValorHora;

Salario = HorasTrabalhadas*ValorHora;

printf("NUMBER = %d\n",NumeroFuncionario);
printf("SALARY = U$ %0.2lf\n",Salario);

    
	return 0;
}

File: beecrowd_urionlinejudge/uri1008Salario.cs
    using System;
    
    class URI {
    
        static void Main(string[] args) { 

		int NumeroFuncionario,HorasTrabalhadas;
		double ValorHora,Salario;
    

        NumeroFuncionario=int.Parse(Console.ReadLine());
        HorasTrabalhadas=int.Parse(Console.ReadLine());
        ValorHora=float.Parse(Console.ReadLine());
    
		Salario = HorasTrabalhadas*ValorHora;

            System.Console.WriteLine(String.Format( "NUMBER = {0}",NumeroFuncionario));
            System.Console.WriteLine(String.Format( "SALARY = U$ {0:0.00}",Salario));
                
        }
    
    }


